db/migrate/20221026150443_add_users_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] Please add the foreign key between likes and users tables as `author_id`, not `user_id` as mentioned on the ERD table.

![image](https://user-images.githubusercontent.com/68796232/198085576-049b3f2e-7400-406d-9b28-829c11cd2fd7.png)

[Hint] You can do it like that:
```ruby
add_reference :likes, :author, foreign_key: { to_table: :users }, null: false
```